proposals/lip-0045.md

```diff
@@ -218,7 +218,7 @@ In this section, we specify the substores that are part of the Interoperability
 | `EMPTY_BYTES` | bytes | "" | The empty byte string. |
 | `EMPTY_FEE_ADDRESS` | bytes | "" | The empty byte string. |
 | `THRESHOLD_MAINCHAIN` | uint32 | `68` | The certificate threshold used for Lisk mainchain. Defined in the DPoS module. |
-| `TOKEN_ID_LSK_MAINCHAIN` | bytes | 0x0000000000000000 | The token ID of the Lisk token. Defined in the Token module. |
+| `TOKEN_ID_LSK` | bytes | `CHAIN_ID_MAINCHAIN` +  0x00000000 | The token ID of the Lisk token. Defined in the Token module. |
```

Comment:
Why do we specify the value here rather than linking to LIP51 like we do in other LIPs?